scripts/deployASB.sh

```diff
@@ -0,0 +1,564 @@
+#!/bin/bash
+
+function collectInputParameters()
+{
+  echo $1
+  if [ -z $1 ]; then
```

Comment:
Looks like you can use the command in codespaces as long as you log in without using device code - need to add documentation that this script should be run outside of browser and explain how to do the az login step, then we can add the cluster admin group step to the main script instead of having to call in an external one.   The name of the cluster admin group though, is something that should be requested as a prerequisite to starting this process.